server/routes/prisonerSearch/index.ts

```diff
@@ -0,0 +1,18 @@
+import express, { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+
+import PrisonerSearchRoutes from './prisonerSearch'
+
+export default function MovingPrisonerRoutes(): Router {
```

Comment:
**Not blocking**

Do we have linting around casing for things?

I think generally in TS, functions are lower camel case and classes/types are upper camel case

As I say - not a blocking comment though 😄